src/main/java/com/epam/jconference/controller/LectureController.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.jconference.controller;
+
+import com.epam.jconference.controller.dto.LectureDto;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+public class LectureController {
+
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping("/lectures/create")
+    public LectureDto create(@RequestBody LectureDto lectureDto) {
```

Comment:
The same like in EventController, try to use @RequestMapping("/lectures") instead